.github/automation/.drone.yml

```diff
@@ -1,5 +1,6 @@
 # *******************************************************************************
 # Copyright 2020 Arm Limited and affiliates.
+# Copyright 2020 FUJITSU LIMITED
```

Comment:
(random spot)

Hopefully the last concern from my side :)

Let's reduce the number of commits (50+ is definitely an overkill), since we use linear history and try to make each commit self-contained and "correct" (meaning that at each commit the library, ideally, should be at least build-able).

Looking at the patch overall I think the commits should be organized as follows (in order):
1. temporary commit: `cpu: x64: silence array-bounds gcc warning`. Will be dropped before the actual merge once we update `github/master` branch (within few days).
2. applying changes to `tests/benchdnn/conv/conv.cpp`
3. moving `set_before_first_get_setting_t` from `src/cpu/x64/cpu_isa_traits.cpp` to `src/common/utils.hpp`
4. commit: `build: add DNNL_TARGET_EMULATOR support`
5. changes to `src/cpu/aarch64/CMakeLists.txt` to enable future extensions to `aarch64` directory
6. adding xbyak_aarch64
7. adding `cpu_isa_traits.{ch}pp`, `jit_generator.hpp`, `jit_utils/*.*` in `src/cpu/aarch64/`, and updating `src/cpu/platform.cpp` 
8. adding JIT reorder, and updating `src/cpu/cpu_reorder.cpp`
9. finally, adding testing for Armv8-A + 512-bit SVE (essentially all remaining changes in `.github/` directory).

I hope it would be fairly easy to change the history this way, because there should not be any conflicts: the commits don't have intersection in files.

Reply:
Thanks for the advice. I'll try it.

Reply:
I corrected the commit logs like the below.
https://github.com/kawakami-k/oneDNN/commits/master_pr_20201009_rc4
Can I force-push this commit logs to fujitsu/master_pr_20201009?

Reply:
Thank you! The changes look good to me :)
Sure, feel free to force-push them here.

Reply:
Hmm, something went wrong -- the PR now has 46 commits, and 8 (9)...